docs/commands/ts.add.md

```diff
@@ -94,22 +99,18 @@ The complexity of `TS.ADD` is always `O(M)`, where `M` is the number of compacti
 
 <details open><summary><b>Append a sample to a temperature time series</b></summary>
 
-Create a temperature time series.
+Create a temperature time series and append a sample.
 
 {{< highlight bash >}}
-127.0.0.1:6379> TS.CREATE temperature:2:32 RETENTION 60000 DUPLICATE_POLICY MAX LABELS sensor_id 2 area_id 32
-OK
+127.0.0.1:6379> TS.ADD temperature:3:11 1548149183000 27 RETENTION 3600
+(integer) 1548149183000
 {{< / highlight >}}
 
-Append a sample to the time series.
+Take the time stamp from the server clock.
```

Comment:
Add a sample to the time series, setting the sample's timestamp according to the server clock.